fontbe/src/paths.rs

```diff
@@ -45,7 +45,8 @@ impl Paths {
         match id {
             WorkId::Features => self.build_dir.join("features.ttf"),
             WorkId::Glyph(name) => self.glyph_file(name.as_str()),
-            WorkId::GlyphMerge => self.build_dir.join("glyf.ttf"),
+            WorkId::GlyphMerge(GlyphMerge::Glyf) => self.build_dir.join("glyf.ttf"),
```

Comment:
I would not call the built file for glyf table "glyf.ttf" because that suggests it is a semi-complete font with sfnt structure, whereas in fact this only contains the blob for one (glyf) table, without sfnt header nor the other tables which would be required to parse glyf (i.e. maxp and loca at the very least).
On the other hand, the "features.ttf" file that fea-rs produces _can_ be decompiled with fontTools/ttx, because in addition to GPOS/GSUB it has a maxp).

Maybe since glyf and loca always go together (they are really _one_ table), you could dump both in the same glyf.ttf file, add an sfnt header to locate them, and also add a minimal maxp table just to know how many glyphs are in loca and what loca format it is. This would be enough for fontTools to able to parse, and can be handy if one wants to quickly inspect the build byproducts (`ttx build/glyf.ttf` currently gives "Unknown file type: 'build/glyf.ttf'")

Alternatively if you think it's preferable to serialize the two to different files, give a different file extension to glyf so that it's clear that it's not a font, not even a partial one, but just one table (glyf.table or glyf.sfnt?)